### PR TITLE
fix tensorflow.go, option.go

### PR DESCRIPTION
**Pull request by datelier**
Signed-off-by: datelier <57349093+datelier@users.noreply.github.com>

### Environment:

- Golang Version: 1.13.5
- Docker Version: 19.03.5
- Kubernetes Version: 1.16.3
- NGT Version: 1.8.4

### Types of changes:

- [ ] Bug fix [type/bug]
- [x] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [x] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensure all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: internal/core/converter/tensorflow/tensorflow.go [hunk 1]
```diff
@@ -26,7 +26,9 @@ type SessionOptions = tf.SessionOptions
 type Operation = tf.Operation
 
 type TF interface {
-	GetVector(feeds []Feed, fetches []Fetch, targets ...*Operation) (values [][][]float64, err error)
+	GetVector(feeds []Feed, fetch Fetch, targets ...*Operation) ([]float64, error)
```
Comment (kpango) [inline internal/core/converter/tensorflow/tensorflow.go]:
```suggestion
	GetVector(interface{}) ([]float64, error)
```

Comment (datelier) [reply internal/core/converter/tensorflow/tensorflow.go]: :+1:

File: internal/core/converter/tensorflow/tensorflow.go [hunk 2]
```diff
@@ -26,31 +26,35 @@ type SessionOptions = tf.SessionOptions
...
+	outputIndex   int
 }
 
-type Fetch struct {
-	OperationName string
-	OutputIndex   int
-}
+const (
+	TWO_DIM   int8 = 2
+	THREE_DIM int8 = 3
```
Comment (kpango) [inline internal/core/converter/tensorflow/tensorflow.go:56]: In golang const value we shouldn't use snake case please use camel case, and I recommend below
```suggestion
	NoDim uint8 = iota
	OneDim
	TwoDim
	ThreeDim
```

**Pull request merged**